students/medrek_tomasz/lesson_03_functions/exponentation.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+
+def exponentiation(a, n):
```

Comment:
FYI:
return 1 if n == 0 else a * exponentiation(a, n - 1)

But propably my solution will be unreadable, so this is only TIP for You

Reply:
Nice tip, didn't know such thing is allowed, thanks.